Gifski/MainWindowController.swift

```diff
@@ -9,6 +9,18 @@ final class MainWindowController: NSWindowController {
 		window?.contentViewController is ConversionViewController
 	}
 
+	private func showWelcomeScreen() {
+		guard App.isFirstLaunch else {
+			return
+		}
+
+		NSAlert.showModal(
+			for: window,
+			message: "Welcome to Gifski!",
+			informativeText: "If you have any feedback, bug reports, or feature requests, kindly use the “Send Feedback” button in the “Help” menu. We will respond to all submissions and it is likely that those reported issues will be fixed immediately.\n\nPlease note that it is preferable to submit a bug report through the feedback button than as an App Store review since the review will not allow us to contact you for more information regarding the bug."
```

Comment:
```suggestion
			informativeText: "If you have any feedback, bug reports or feature requests, kindly use the “Send Feedback” button in the “Help” menu. We will respond to all submissions and it is likely that reported issues will be fixed immediately.\n\nPlease note that it is preferable to submit a bug report through the feedback button than as an App Store review since the review will not allow us to contact you for more information regarding the bug."
```

I think this is a very descriptive paragraph, nice job!

Reply:
I like the Oxford comma: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

Reply:
👍